ldregistry/ui/assets/js/app/qconfig.js

```diff
@@ -1,7 +1,7 @@
       // configuration
       var config = {
         endpoints: {
-          "default": "/registry/system/query",
+          "default": "/system/query",
```

Comment:
Deploying at the domain root instead of under some context path is unusual, at least for us. The root domain is typically shared across others services, not just the registry. Which is why the config-base supports this with a default context path. 

I would rather leave this in place to avoid impact on other inheritors of `registry-config-base` unless it is causing a real issue.